docs/src/adoc/index.adoc

```diff
@@ -1211,7 +1211,7 @@ Reflection mappers support the following annotations. Unless otherwise noted, al
 * `@PropagateNull` on a given attribute propagates a null value for a specific attribute up. So if the attribute or column is null, the whole bean will be discarded and a null value will be used instead of the bean itself. This annotation can also be used on the bean
   class with a column name that must be present in the result set. When used on an attribute, no value must be set.
 * `@Nullable` for `ConstructorMapper` arguments. If no column is mapped onto a specific constructor argument, don't fail but use `null` as value. Any annotation called `Nullable` is accepted.
-* `@Unmappable` for `BeanMapper` bean properties and `FieldMapper` fields. Any attribute with this annotation will be ignored.
+* `@JdbiProperty(map=false, bind=false)` for `BeanMapper` bean properties and `FieldMapper` fields. Any attribute with this annotation will be ignored for binding and mapping respectively.
```

Comment:
this is a very terse explanation and it is explained in context of "Bean mapping".  This should focus exclusively on this (not talk about `bind=true|false`). 

Unless someone knows where to look, no one is going to find this if they look for ways to suppress binding. It would be good to have some explanation at the end of section 4.3.4 (http://jdbi.org/#_supported_argument_types) that talks more about using this annotation to control binding (`@JdbiProperty(bind=false)`)